Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:8
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package*.json ./
+
+RUN npm install
```

Comment:
To run the app, you don't need the modules listed in the `devDependencies`. You only need the ones list in `dependencies` in the _package.json_. When you run `npm install` without `NODE_ENV` set to "production" it installs all dependencies and devDependencies as `NODE_ENV` defaults to "development".

Instead, ensure that you set `NODE_ENV=production` when running `npm install` so your _./node_modules_ folder in your docker image is optimized in size as it will only have the modules listed in `dependencies`.

```
NODE_ENV=production npm install
```

Reply:
Thanks! Good to know.